x-pack/plugins/security_solution/public/management/pages/policy/view/ingest_manager_integration/endpoint_package_custom_extension/components/styled_components.tsx

```diff
@@ -14,11 +14,11 @@ export const StyledEuiFlexGridGroup = styled(EuiFlexGroup)`
 `;
 
 export const StyledEuiFlexGridItem = styled(EuiFlexItem)<{
-  gridArea: string;
-  alignItems?: string;
+  gridarea: string;
```

Comment:
Out of curiosity? Why was a change to flat case needed?

Reply:
To avoid this react warning:
```
Warning: React does not recognize the `alignItems` prop on a DOM element. If you intentionally want it to appear in the DOM as a custom attribute, spell it as lowercase `alignitems` instead. If you accidentally passed it from a parent component, remove it from the DOM element.
```